src/GafferScene/Light.cpp

```diff
@@ -64,6 +65,7 @@ Light::Light( const std::string &name )
 	addChild( new Plug( "parameters" ) );
 	addChild( new BoolPlug( "defaultLight", Gaffer::Plug::Direction::In, true ) );
 	addChild( new FloatPlug( "visualiserScale", Gaffer::Plug::Direction::In, 1.0 ) );
+	addChild( new BoolPlug( "visualiserShaded", Gaffer::Plug::Direction::In, true ) );
```

Comment:
Def want to do that, iirc, we agreed to leave this PR as-is and fix this, and visualiser scale in another PR before release, to avoid this getting too much of a behemoth.